### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by PvlSport**

### CODE REVIEW

File: fx/src/main/resources/org/unigrid/janus/view/settings.fxml [hunk 1]
```diff
@@ -249,7 +249,7 @@
                               <Label layoutY="27.0" text="Create a backup of your wallet to store in case your computer breaks down or is lost." textFill="#ccccff" wrapText="true" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" />
                               <HBox layoutY="65.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0">
                                  <children>
-                                    <Button mnemonicParsing="false" onMouseClicked="#onBackupWallet" style="-fx-cursor: hand;" text="Backup" />
+                                    <Button fx:id="btnSetExportBackup" mnemonicParsing="false" onMouseClicked="#onBackupWallet" text="Backup" />
```
Comment (adam-waldenberg) [inline fx/src/main/resources/org/unigrid/janus/view/settings.fxml:252]: This name can't change, then it wont attach to the onBackupWallet() action anymore in SettingsController. We also want to use the onSomeName() scheme for input events.

File: fx/src/test/java/org/unigrid/janus/controller/SettingsControllerTest.java [hunk 2]
```diff
@@ -173,4 +178,19 @@ public void shouldChangePassword() {
...
+
+	@Example
+	public void shouldAskForFile() {
+		new FileChooserMockUp();
+
+		robot.clickOn("#btnSettings");
+		robot.clickOn("#btnSetExport");
+
+		robot.clickOn("#btnSetExportImport");
+		robot.clickOn("#btnSetExportBackup");
```
Comment (adam-waldenberg) [inline fx/src/test/java/org/unigrid/janus/controller/SettingsControllerTest.java:190]: So this needs to be changed to onBackupWallet() as well.

**Review by adam-waldenberg — CHANGES_REQUESTED** [review]
Get rid of AddressTest .. The only testable logic in there is the toString().

Get rid of UpdateURLTest and BootStrapModel test as well - they are not really doing anything.

File: fx/src/test/java/org/unigrid/janus/controller/SettingsControllerTest.java (hunk 2 above)
Comment (PvlSport) [reply fx/src/test/java/org/unigrid/janus/controller/SettingsControllerTest.java:190]: do you mean changing to `robot.clickOn("#onBackupWallet");` ?

**Pull request merged**